fission/logdb/influxdb.go

```diff
@@ -17,47 +17,35 @@ limitations under the License.
 package logdb
 
 import (
-	"log"
-	"time"
-
+	"encoding/json"
 	"fmt"
-
+	"log"
+	"net/http"
+	"net/url"
 	"strings"
+	"time"
 
 	influxdbClient "github.com/influxdata/influxdb/client/v2"
+
+	"github.com/fission/fission"
 )
 
 const (
 	INFLUXDB_DATABASE = "fissionFunctionLog"
+	INFLUXDB_URL      = "http://influxdb:8086/query"
 )
 
-func NewInfluxDB(cnf DBConfig) (InfluxDB, error) {
-	dbClient, err := influxdbClient.NewHTTPClient(influxdbClient.HTTPConfig{
-		Addr:     cnf.Endpoint,
-		Username: cnf.Username,
-		Password: cnf.Password,
-	})
-	if err != nil {
-		log.Fatal(err)
-		return InfluxDB{}, err
-	}
-
-	return InfluxDB{
-		dbClient: dbClient,
-	}, nil
+func NewInfluxDB(serverURL string) (InfluxDB, error) {
+	return InfluxDB{endpoint: serverURL}, nil
```

Comment:
Can't we use the influxdb client, but specify the new URL in the HTTPConfig Addr field?  That way we won't have to implement the query() function too.

(Looking at the code https://github.com/influxdata/influxdb/blob/master/client/v2/client.go#L82, it looks like it should just work.)